### ISSUE TITLE
Update Node.js from 16 to 20, due to deprecation

### ISSUE THREAD

**Issue by paoloverdini97** [NuGet/setup-nuget#112]
Node.js 16 actions are deprecated. Please update the action to use Node.js 20, according to https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20

**Comment by watfordsuzy** [comment]
I saw #59 was merged back in October, could we get that bumped as a release?

**Comment by i2van** [comment]
As a workaround latest commit can be used:

```yaml
    - name: Setup NuGet
      uses: NuGet/setup-nuget@f685ada866d3ce9775950304dccf4cec8208e6cf
```

**Comment by jeffkl** [comment]
I've released a new version, 1.3.0 and updated the tag for v1 to point to it.

**Comment by jeffkl** [comment]
I've had to change the release tag to `v2`, `v2.0`, or `v2.0.0` instead
